samtranslator/model/api/api_generator.py

```diff
@@ -555,6 +555,24 @@ def _openapi_postprocess(self, definition_body):
                 components['schemas'] = definition_body['definitions']
                 definition_body['components'] = components
                 del definition_body['definitions']
+
```

Comment:
Please either add a comment here about what this section is trying to do or have this method call other methods with docstrings that explain what things we're changing and why.

Reply:
I have added comments for the other if statements but adding at the top level makes more sense. Thank you! I have updated with comments for this part